### PR TITLE
Salesforce Unify Direct Integration GA [DOC-960]

### PR DESCRIPTION
**Pull request by pwseg**
### Proposed changes

- Added new content for the Salesforce Unify Direct Integration.

### Merge timing

- 27 August 2024.

### CODE REVIEW

File: src/unify/salesforce-unify.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+This guide outlines the process for setting up Salesforce as a data source with Segment Profiles. 
+
+Once configured, this integration lets you send Salesforce data directly to Segment Profiles, eliminating the need for a data warehouse and enabling faster data synchronization and profile enrichment.
+
+## Prerequisites
+
+Before you begin, make sure that you have the following:
+
+- a Segment workspace with [Unify](/docs/unify/) enabled and [Identity Resolution](/docs/unify/identity-resolution/) set up
+- administrator access to your Salesforce account
```
Comment (forstisabella) [inline src/unify/salesforce-unify.md:15]:
```suggestion
- Administrator access to your Salesforce account
```

File: src/unify/salesforce-unify.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+5. Name your model and save it.
+
+### 4. Configure mapping
+
+1. In the Salesforce source, navigate to **Models > Add mapping**.
+2. Segment returns you to the Segment Profiles Destination. Click **Add mapping**.
+2. Select your data model and map your selected Salesforce data to fields in Segment Profiles (typically "Identify" for profile updates).
+   - Select the Profile space you want to update.
+   - Map Salesforce fields to Segment Profile fields.
+   - **You must map either a User ID or Anonymous ID field.**
```
Comment (forstisabella) [inline src/unify/salesforce-unify.md:55]:
```suggestion
2. Select your data model and map your selected Salesforce data to fields in Segment Profiles (typically "Identify" for profile updates).
   - Select the Profile space you want to update.
   - Map Salesforce fields to Segment Profile fields. **You must map either a User ID or Anonymous ID field.**
```

File: src/unify/salesforce-unify.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+4. Test your mapping with real data from Salesforce.
+5. Save your mapping configuration.
+
+![A screenshot of the "Select Mappings" interface in Segment, showing fields for mapping event data from a source to a destination. The interface includes options for selecting objects, customizing objects, and mapping fields such as Profile Space, User ID, Anonymous ID, Group ID, Traits, and Timestamp.](./images/select_mappings.png)
```
Comment (forstisabella) [inline src/unify/salesforce-unify.md:59]: 🔥

File: src/unify/salesforce-unify.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+1. From the overview tab of the Segment Profiles destination you created, toggle the **Mapping Status** switch to **Enabled**.
+2. Return to your Salesforce source and navigate to **Settings > Basic settings.**
+2. Toggle the **Enable source** switch to on.
+
+Data now begins syncing between Salesforce and Segment.
+
+## Data synchronization
+
+After you've connected Salesforce and the Segment Profiles destination, the integration begin syncing data based on the following:
```
Comment (forstisabella) [inline src/unify/salesforce-unify.md:73]:
```suggestion
After you've connected Salesforce and the Segment Profiles destination, the integration begins syncing data:
```

File: src/unify/salesforce-unify.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+- The initial sync includes historical data within the range specified in your Selective Sync settings.
+- Further syncs take place at regular intervals (typically hourly).
+
+## Best practices
+
+Keep the following in mind as you work with the Salesforce Unify Direct Integration:
+
+- Start with a subset of data to test your integration before expanding to all Salesforce objects.
+- Regularly review your mapping to ensure it reflects any changes in your Salesforce schema or Segment Profile needs.
+- Monitor your integration for any errors or data discrepancies.
```
Comment (forstisabella) [inline src/unify/salesforce-unify.md:85]: which integration? The Salesforce source or the Segment Profiles destination (or both, maybe?)

Comment (pwseg) [reply src/unify/salesforce-unify.md]: Yeah, good callout.  Just clarified that.

**Review by forstisabella — APPROVED** [review]

**Pull request merged**